[0.9] Server blows up, probably because a bad query..

```
[2] $ influxd                                                                                                                                                                   ✘

 8888888           .d888 888                   8888888b.  888888b.
   888            d88P"  888                   888  "Y88b 888  "88b
   888            888    888                   888    888 888  .88P
   888   88888b.  888888 888 888  888 888  888 888    888 8888888K.
   888   888 "88b 888    888 888  888  Y8bd8P' 888    888 888  "Y88b
   888   888  888 888    888 888  888   X88K   888    888 888    888
   888   888  888 888    888 Y88b 888 .d8""8b. 888  .d88P 888   d88P
 8888888 888  888 888    888  "Y88888 888  888 8888888P"  8888888P"

GOMAXPROCS set to 4
no configuration provided, using default settings
2015/06/26 11:00:28 [INFO] raft: Node at 127.0.0.1:8088 [Follower] entering Follower state
2015/06/26 11:00:28 read local node id: 1
2015/06/26 11:00:30 [WARN] raft: Heartbeat timeout reached, starting election
2015/06/26 11:00:30 [INFO] raft: Node at 127.0.0.1:8088 [Candidate] entering Candidate state
2015/06/26 11:00:30 [DEBUG] raft: Votes needed: 1
2015/06/26 11:00:30 [DEBUG] raft: Vote granted. Tally: 1
2015/06/26 11:00:30 [INFO] raft: Election won. Tally: 1
2015/06/26 11:00:30 [INFO] raft: Node at 127.0.0.1:8088 [Leader] entering Leader state
2015/06/26 11:00:30 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/06/26 11:00:30 [DEBUG] raft: Node 127.0.0.1:8088 updated peer set (2): [127.0.0.1:8088]
2015/06/26 11:00:30 [DEBUG] raft: Node 127.0.0.1:8088 updated peer set (2): [127.0.0.1:8088]
2015/06/26 11:00:30 [DEBUG] raft: Node 127.0.0.1:8088 updated peer set (2): [127.0.0.1:8088]
2015/06/26 11:00:30 [DEBUG] raft: Node 127.0.0.1:8088 updated peer set (2): [127.0.0.1:8088]
2015/06/26 11:00:30 [DEBUG] raft: Node 127.0.0.1:8088 updated peer set (2): [127.0.0.1:8088]
2015/06/26 11:00:30 [DEBUG] raft: Node 127.0.0.1:8088 updated peer set (2): [127.0.0.1:8088]
[httpd] 2015/06/26 11:00:30 listening on HTTP: [::]:8086
2015/06/26 11:00:30 Sending anonymous usage statistics to m.influxdb.com
[http] 2015/06/26 11:00:34 127.0.0.1 - sms [26/Jun/2015:11:00:34 +0200] POST /write?db=sms HTTP/1.1 204 0 - Hurley v0.1 cdf6492a-1be1-11e5-8001-000000000000 966.994µs
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x26b76f]

goroutine 51 [running]:
github.com/influxdb/influxdb/influxql.(*SelectStatement).RewriteDistinct(0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/influxql/ast.go:780 +0x37f
github.com/influxdb/influxdb/tsdb.(*QueryExecutor).rewriteSelectStatement(0xc208035c00, 0x0, 0xd95180, 0x0, 0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/tsdb/query_executor.go:271 +0x18a
github.com/influxdb/influxdb/tsdb.(*QueryExecutor).executeSelectStatement(0xc208035c00, 0x0, 0xc2080ce180, 0x0, 0x0, 0xc2080de480, 0x2710, 0x0, 0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/tsdb/query_executor.go:219 +0x58
github.com/influxdb/influxdb/tsdb.func·004()
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/tsdb/query_executor.go:157 +0x727
created by github.com/influxdb/influxdb/tsdb.(*QueryExecutor).ExecuteQuery
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/tsdb/query_executor.go:211 +0x22b

goroutine 1 [chan receive (nil chan)]:
main.(*Main).Run(0xc2080a7f60, 0xc20800a000, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/cmd/influxd/main.go:81 +0x450
main.main()
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/cmd/influxd/main.go:39 +0x1d0

goroutine 6 [syscall]:
os/signal.loop()
	/usr/local/Cellar/go/1.4.2/libexec/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/local/Cellar/go/1.4.2/libexec/src/os/signal/signal_unix.go:27 +0x35

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/asm_amd64.s:2232 +0x1

goroutine 8 [IO wait]:
net.(*pollDesc).Wait(0xc208011bf0, 0x72, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_poll_runtime.go:84 +0x47
net.(*pollDesc).WaitRead(0xc208011bf0, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_poll_runtime.go:89 +0x43
net.(*netFD).accept(0xc208011b90, 0x0, 0xd8dca0, 0xc2080e6000)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_unix.go:419 +0x40b
net.(*TCPListener).AcceptTCP(0xc2080381a8, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/tcpsock_posix.go:234 +0x4e
net.(*TCPListener).Accept(0xc2080381a8, 0x0, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/tcpsock_posix.go:244 +0x4c
github.com/influxdb/influxdb/tcp.(*Mux).Serve(0xc2080b8b10, 0xd8ebd8, 0xc2080381a8, 0x0, 0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/tcp/mux.go:45 +0x57
created by github.com/influxdb/influxdb/cmd/influxd/run.func·003
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:274 +0x762

goroutine 9 [chan receive]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc2080381b0, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x80
github.com/influxdb/influxdb/meta.(*raftLayer).Accept(0xc2080b8b40, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/meta/store.go:1636 +0x64
github.com/hashicorp/raft.(*NetworkTransport).listen(0xc208011c70)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/net_transport.go:346 +0x50
created by github.com/hashicorp/raft.NewNetworkTransport
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/net_transport.go:138 +0x2c4

goroutine 10 [select]:
github.com/hashicorp/raft.(*Raft).leaderLoop(0xc20800c380)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/raft.go:807 +0xc7c
github.com/hashicorp/raft.(*Raft).runLeader(0xc20800c380)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/raft.go:779 +0x74e
github.com/hashicorp/raft.(*Raft).run(0xc20800c380)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/raft.go:576 +0xc9
github.com/hashicorp/raft.*Raft.(github.com/hashicorp/raft.run)·fm()
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/raft.go:241 +0x27
github.com/hashicorp/raft.func·011()
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/state.go:152 +0x51
created by github.com/hashicorp/raft.(*raftState).goFunc
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/state.go:153 +0xe3

goroutine 11 [select]:
github.com/hashicorp/raft.(*Raft).runFSM(0xc20800c380)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/raft.go:490 +0xf74
github.com/hashicorp/raft.*Raft.(github.com/hashicorp/raft.runFSM)·fm()
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/raft.go:242 +0x27
github.com/hashicorp/raft.func·011()
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/state.go:152 +0x51
created by github.com/hashicorp/raft.(*raftState).goFunc
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/state.go:153 +0xe3

goroutine 12 [select]:
github.com/hashicorp/raft.(*Raft).runSnapshots(0xc20800c380)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/raft.go:1594 +0x466
github.com/hashicorp/raft.*Raft.(github.com/hashicorp/raft.runSnapshots)·fm()
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/raft.go:243 +0x27
github.com/hashicorp/raft.func·011()
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/state.go:152 +0x51
created by github.com/hashicorp/raft.(*raftState).goFunc
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/hashicorp/raft/state.go:153 +0xe3

goroutine 18 [chan receive]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc2080381b8, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x80
github.com/influxdb/influxdb/meta.(*Store).serveExecListener(0xc208076640)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/meta/store.go:436 +0x8f
created by github.com/influxdb/influxdb/meta.(*Store).Open
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/meta/store.go:183 +0x1c2

goroutine 19 [select]:
github.com/influxdb/influxdb/cmd/influxd/run.(*Server).monitorErrorChan(0xc2080ce000, 0xc20802c5a0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:390 +0x14a
created by github.com/influxdb/influxdb/cmd/influxd/run.func·003
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:280 +0x8d3

goroutine 14 [select]:
github.com/influxdb/influxdb/services/hh.(*Service).retryWrites(0xc20804b0e0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/hh/service.go:98 +0x2e7
created by github.com/influxdb/influxdb/services/hh.(*Service).Open
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/hh/service.go:62 +0xef

goroutine 15 [select]:
github.com/influxdb/influxdb/services/hh.(*Service).expireWrites(0xc20804b0e0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/hh/service.go:116 +0x2a4
created by github.com/influxdb/influxdb/services/hh.(*Service).Open
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/hh/service.go:63 +0x109

goroutine 16 [chan receive]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc2080381c0, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x80
github.com/influxdb/influxdb/cluster.(*Service).serve(0xc208011a40)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/cluster/service.go:79 +0x102
created by github.com/influxdb/influxdb/cluster.(*Service).Open
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/cluster/service.go:56 +0x67

goroutine 34 [select]:
github.com/influxdb/influxdb/services/precreator.(*Service).runPrecreation(0xc20802ed20)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/precreator/service.go:72 +0x407
created by github.com/influxdb/influxdb/services/precreator.(*Service).Open
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/precreator/service.go:50 +0xcb

goroutine 35 [chan receive]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc2080381c8, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x80
github.com/influxdb/influxdb/services/snapshotter.(*Service).serve(0xc20802ee10)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/snapshotter/service.go:68 +0xad
created by github.com/influxdb/influxdb/services/snapshotter.(*Service).Open
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/snapshotter/service.go:46 +0x62

goroutine 36 [IO wait]:
net.(*pollDesc).Wait(0xc20812ea70, 0x72, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_poll_runtime.go:84 +0x47
net.(*pollDesc).WaitRead(0xc20812ea70, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_poll_runtime.go:89 +0x43
net.(*netFD).accept(0xc20812ea10, 0x0, 0xd8dca0, 0xc20800a950)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_unix.go:419 +0x40b
net.(*TCPListener).AcceptTCP(0xc20806c048, 0xc2080e2380, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/tcpsock_posix.go:234 +0x4e
net.(*TCPListener).Accept(0xc20806c048, 0x0, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/tcpsock_posix.go:244 +0x4c
net/http.(*Server).Serve(0xc2080de060, 0xd8ebd8, 0xc20806c048, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1728 +0x92
net/http.Serve(0xd8ebd8, 0xc20806c048, 0xd947a0, 0xc20800a940, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1606 +0xa1
github.com/influxdb/influxdb/services/admin.(*Service).serve(0xc2080b8900)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/admin/service.go:71 +0x147
created by github.com/influxdb/influxdb/services/admin.(*Service).Open
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/admin/service.go:39 +0xd9

goroutine 37 [select]:
github.com/influxdb/influxdb/services/continuous_querier.(*Service).backgroundLoop(0xc208011ab0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/continuous_querier/service.go:151 +0x1fc
created by github.com/influxdb/influxdb/services/continuous_querier.(*Service).Open
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/continuous_querier/service.go:87 +0x1ed

goroutine 38 [IO wait]:
net.(*pollDesc).Wait(0xc20812eae0, 0x72, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_poll_runtime.go:84 +0x47
net.(*pollDesc).WaitRead(0xc20812eae0, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_poll_runtime.go:89 +0x43
net.(*netFD).accept(0xc20812ea80, 0x0, 0xd8dca0, 0xc20800b428)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_unix.go:419 +0x40b
net.(*TCPListener).AcceptTCP(0xc20806c050, 0xc208058dc8, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/tcpsock_posix.go:234 +0x4e
net.(*TCPListener).Accept(0xc20806c050, 0x0, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/tcpsock_posix.go:244 +0x4c
net/http.(*Server).Serve(0xc208106480, 0xd8ebd8, 0xc20806c050, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1728 +0x92
net/http.Serve(0xd8ebd8, 0xc20806c050, 0xd94750, 0xc208011b20, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1606 +0xa1
github.com/influxdb/influxdb/services/httpd.(*Service).serve(0xc208035c80)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/service.go:83 +0x79
created by github.com/influxdb/influxdb/services/httpd.(*Service).Open
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/service.go:51 +0x283

goroutine 39 [select]:
github.com/influxdb/influxdb/services/retention.(*Service).deleteShardGroups(0xc20802c960)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/retention/service.go:67 +0x433
created by github.com/influxdb/influxdb/services/retention.(*Service).Open
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/retention/service.go:44 +0x67

goroutine 40 [select]:
github.com/influxdb/influxdb/services/retention.(*Service).deleteShards(0xc20802c960)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/retention/service.go:101 +0x821
created by github.com/influxdb/influxdb/services/retention.(*Service).Open
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/retention/service.go:45 +0x81

goroutine 41 [chan receive]:
github.com/influxdb/influxdb/cmd/influxd/run.(*Server).startServerReporting(0xc2080ce000)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:341 +0x5e
created by github.com/influxdb/influxdb/cmd/influxd/run.func·003
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:303 +0xd78

goroutine 42 [select]:
github.com/influxdb/influxdb/cmd/influxd/run.(*Command).monitorServerErrors(0xc20802c360)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/cmd/influxd/run/command.go:118 +0x2da
created by github.com/influxdb/influxdb/cmd/influxd/run.(*Command).Run
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/cmd/influxd/run/command.go:102 +0xa1a

goroutine 21 [IO wait]:
net.(*pollDesc).Wait(0xc20810c060, 0x72, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_poll_runtime.go:84 +0x47
net.(*pollDesc).WaitRead(0xc20810c060, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_poll_runtime.go:89 +0x43
net.(*netFD).Read(0xc20810c000, 0xc2080fc000, 0x1000, 0x1000, 0x0, 0xd8dca0, 0xc20800a9a0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_unix.go:242 +0x40f
net.(*conn).Read(0xc208038030, 0xc2080fc000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/net.go:121 +0xdc
net/http.noteEOFReader.Read(0xd948a8, 0xc208038030, 0xc208048058, 0xc2080fc000, 0x1000, 0x1000, 0xc2081560c0, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/transport.go:1270 +0x6e
net/http.(*noteEOFReader).Read(0xc208158140, 0xc2080fc000, 0x1000, 0x1000, 0xc208014400, 0x0, 0x0)
	<autogenerated>:125 +0xd4
bufio.(*Reader).fill(0xc20802d2c0)
	/usr/local/Cellar/go/1.4.2/libexec/src/bufio/bufio.go:97 +0x1ce
bufio.(*Reader).Peek(0xc20802d2c0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.4.2/libexec/src/bufio/bufio.go:132 +0xf0
net/http.(*persistConn).readLoop(0xc208048000)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/transport.go:842 +0xa4
created by net/http.(*Transport).dialConn
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/transport.go:660 +0xc9f

goroutine 22 [select]:
net/http.(*persistConn).writeLoop(0xc208048000)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/transport.go:945 +0x41d
created by net/http.(*Transport).dialConn
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/transport.go:661 +0xcbc

goroutine 50 [chan receive]:
github.com/influxdb/influxdb/services/httpd.(*Handler).serveQuery(0xc208011b20, 0xd94d30, 0xc2081242c0, 0xc20812c750, 0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/handler.go:241 +0x7c4
github.com/influxdb/influxdb/services/httpd.*Handler.(github.com/influxdb/influxdb/services/httpd.serveQuery)·fm(0xd94d30, 0xc2081242c0, 0xc20812c750, 0x0)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/handler.go:90 +0x4f
github.com/influxdb/influxdb/services/httpd.func·001(0xd94d30, 0xc2081242c0, 0xc20812c750)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/handler.go:601 +0x78
net/http.HandlerFunc.ServeHTTP(0xc20801f440, 0xd94d30, 0xc2081242c0, 0xc20812c750)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·002(0xd94c58, 0xc208124260, 0xc20812c750)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/handler.go:659 +0x288
net/http.HandlerFunc.ServeHTTP(0xc20800bcf0, 0xd94c58, 0xc208124260, 0xc20812c750)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·003(0xd94c58, 0xc208124260, 0xc20812c750)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/handler.go:668 +0xb0
net/http.HandlerFunc.ServeHTTP(0xc20801f460, 0xd94c58, 0xc208124260, 0xc20812c750)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·004(0xd94c58, 0xc208124260, 0xc20812c750)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/handler.go:701 +0x31a
net/http.HandlerFunc.ServeHTTP(0xc20800bd30, 0xd94c58, 0xc208124260, 0xc20812c750)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·005(0xd94c58, 0xc208124260, 0xc20812c750)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/handler.go:711 +0x179
net/http.HandlerFunc.ServeHTTP(0xc20800bd50, 0xd94c58, 0xc208124260, 0xc20812c750)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·006(0xd94c58, 0xc208124240, 0xc20812c750)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/handler.go:719 +0x123
net/http.HandlerFunc.ServeHTTP(0xc20801f480, 0xd94c58, 0xc208124240, 0xc20812c750)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·007(0xd94c20, 0xc20804a0a0, 0xc20812c750)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/handler.go:729 +0x12c
net/http.HandlerFunc.ServeHTTP(0xc20801f4a0, 0xd94c20, 0xc20804a0a0, 0xc20812c750)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1265 +0x41
github.com/bmizerany/pat.(*PatternServeMux).ServeHTTP(0xc2080380f8, 0xd94c20, 0xc20804a0a0, 0xc20812c750)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/bmizerany/pat/mux.go:109 +0x21c
github.com/influxdb/influxdb/services/httpd.(*Handler).ServeHTTP(0xc208011b20, 0xd94c20, 0xc20804a0a0, 0xc20812c750)
	/private/tmp/influxdb20150611-31144-myuhfv/influxdb-0.9.0/src/github.com/influxdb/influxdb/services/httpd/handler.go:166 +0x256
net/http.serverHandler.ServeHTTP(0xc208106480, 0xd94c20, 0xc20804a0a0, 0xc20812c750)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1703 +0x19a
net/http.(*conn).serve(0xc20804a000)
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1204 +0xb57
created by net/http.(*Server).Serve
	/usr/local/Cellar/go/1.4.2/libexec/src/net/http/server.go:1751 +0x35e
```

--- a/ui/src/pageLayout/containers/Nav.tsx
+++ b/ui/src/pageLayout/containers/Nav.tsx
@@ -140,6 +140,7 @@ class SideNav extends PureComponent<Props, State> {
           )}
           active={getNavItemActivation(['settings'], location.pathname)}
         />
+        <CloudNav />
         <NavMenu.Item
           titleLink={className => (
             <a className={className} href={feedbackLink} target="_blank">
@@ -153,7 +154,6 @@ class SideNav extends PureComponent<Props, State> {
           )}
           active={getNavItemActivation(['feedback'], location.pathname)}
         />
-        <CloudNav />
       </NavMenu>
     )
   }

